pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2250,6 +2250,15 @@ private CompletableFuture<Integer> unloadServiceUnit(NamespaceBundle serviceUnit
         topics.forEach((name, topicFuture) -> {
             TopicName topicName = TopicName.get(name);
             if (serviceUnit.includes(topicName)) {
+                if (ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(pulsar)
+                        && ExtensibleLoadManagerImpl.isInternalTopic(topicName.toString())) {
+                    if (log.isDebugEnabled()) {
```

Comment:
Should we use `ExtensibleLoadManagerImpl.debug` here?

Reply:
Agreed. I think it is better to put this log under `ExtensibleLoadManagerImpl.debug ` too.  Updated.